test/rules/experiments/nonfunction-type-aliases/rules/prefer_mixin.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
```

Comment:
nit: year

Reply:
The approach that we've taken in the past is to keep the copyright from the original since the new contents will get copied back to it when the experiment is finished.